.evergreen/config.yml

```diff
@@ -48,6 +48,14 @@ functions:
           ${compile_env|} ./libmongocrypt/.evergreen/compile.sh
           cd ./libmongocrypt/bindings/java/mongocrypt && ${test_env|} ./.evergreen/test.sh
 
+  "publish snapshot":
+    - command: shell.exec
+      params:
+        silent: true
+        script: |-
+          eval "${inject_env_from_evergreen}"
```

Comment:
After this call `project_directory` is now available and in scope.

Reply:
Ack

Reply:
inject_env_from_evergreen is only for the (now unused AWS credentials). Since I see `nexus_username` as a project variable in the Evergreen project page, it's always accessible with ${nexus_password}`. I can remove all occurrences of `${inject_env_from_evergreen}`, or if you'd like to here, go for it.

Reply:
removed